cos-fleetshard-operator-debezium/src/main/java/org/bf2/cos/fleetshard/operator/debezium/model/DebeziumDataShape.java

```diff
@@ -0,0 +1,54 @@
+package org.bf2.cos.fleetshard.operator.debezium.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class DebeziumDataShape {
+
+    private String keyConverterString;
+    private String valueConverterString;
+    private KafkaConnectConverter keyConverter;
+    private KafkaConnectConverter valueConverter;
+
+    private static KafkaConnectConverter createConverter(String converterType) {
+        switch (converterType) {
+            case "JSON":
+                return new ApicurioJsonConverter();
+            case "AVRO":
+                return new ApicurioAvroConverter();
+            default:
+            case "JSON_WITHOUT_SCHEMA":
+            case "JSON without schema":
```

Comment:
do we need those two additional case here ? they don't hurt but at the same time they are not really meaningful as the same case is covered by the default.

Reply:
Yes. This is only temporarily a default as long as we don't have schema registry support. But the `JSON_WITHOUT_SCHEMA` option will be removed after the next catalog release (where `JSON_WITHOUT_SCHEMA` will change to `JSON without schema`. I think once everything is settled I'll maybe make this an exception eventually?

Reply:
maybe add a comment to explain why we have two explicit cases that led to a default behavior, you know, for the next one that have to understand this case.

also we need to clarify how we support schemes with PM, maybe the registry will be needed anyway when it is about dealing with schemes.

Reply:
@lburgazzoli will revisit this when the catalog is changed. Then we should also have a decision that I put on the list for today's backlog refinement session.